code-studio/src/js/components/GridEditor.jsx

```diff
@@ -0,0 +1,176 @@
+/**
+ * @overview React component to allow for easy editing of Karel Grids.
+ * Used in LevelBuilder, and relies on some apps code for validation.
+ * Supports both Bee and Farmer skins.
+ */
+/* global React, dashboard */
+
+var BeeCell = require('@cdo/apps/src/maze/beeCell');
+var Cell = require('@cdo/apps/src/maze/cell');
+var mazeUtils = require('@cdo/apps/src/maze/mazeUtils');
+
+var BeeCellEditor = require('./BeeCellEditor.jsx');
+var CellEditor = require('./CellEditor.jsx');
+
+window.dashboard = window.dashboard || {};
+
+window.dashboard.GridEditor = (function (React) {
```

Comment:
No need to wrap this anymore, since we now have modules. You can just do
`window.dashboard.GridEditor = React.createClass({`